README.md

```diff
@@ -422,13 +431,13 @@ given URL to perform a read operation.
 Convenience function that invokes an Authorization Capability against a
 given URL to perform a write operation.
 
-**Kind**: instance method of [<code>ZcapClient</code>](#ZcapClient)  
-**Returns**: <code>Promise.&lt;object&gt;</code> - - A promise that resolves to an HTTP response.  
+**Kind**: instance method of [<code>ZcapClient</code>](#ZcapClient)
+**Returns**: <code>Promise.&lt;object&gt;</code> - - A promise that resolves to an HTTP response.
 
 | Param | Type | Description |
 | --- | --- | --- |
 | options | <code>object</code> | The options to use. |
-| options.url | <code>string</code> | The relative URL to invoke the   Authorization Capability against. |
+| options.url | <code>string</code> | The URL to invoke the   Authorization Capability against. |
```

Comment:
```suggestion
| options.url | <code>string</code> | The URL to invoke the Authorization Capability against. |
```

Reply:
Those things were already there -- I'm not going to mess with them now. I don't know if they are from autogeneration code or if they are important or what.